src/components/v5/common/Table/Table.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
+import clsx from 'clsx';
+import { TableProps } from './types';
+import { useMobile } from '~hooks';
+
+const displayName = 'v5.common.Table';
+
+const Table = <T,>({
+  className,
+  tableTitle,
+  columns,
+  fields,
+  burgerColumn,
+  setSelectedRowId,
+}: TableProps<T>) => {
+  const isMobile = useMobile();
+
+  const table = useReactTable<T>({
+    data: fields,
+    columns: burgerColumn ? [...columns, ...burgerColumn] : columns,
+    getCoreRowModel: getCoreRowModel(),
+  });
+
+  const HeaderMobileTable = ({ headerGroups, lastColumn }) =>
+    headerGroups.map((headerGroup) => (
+      <div
+        key={headerGroup.id}
+        className={`flex flex-col bg-gray-50 rounded-t-lg border-gray-200 
+      capitalize text-gray-600 gap-[1.3rem] py-2 text-sm w-[7.5rem] border-r`}
+      >
+        {headerGroup.headers.map((header) => (
+          <div
+            key={header.id}
+            className={clsx(
+              'text-left text-sm text-gray-600 font-normal rounded-t-lg px-4',
+              { hidden: lastColumn.column.id === header.id },
+            )}
+          >
+            {header.isPlaceholder
+              ? null
+              : flexRender(header.column.columnDef.header, header.getContext())}
+          </div>
+        ))}
+      </div>
+    ));
+
+  return (
+    <div className={className}>
+      {tableTitle && <h5 className="text-2 mb-3">{tableTitle}</h5>}
+      <div className="border border-gray-200 rounded-lg">
+        <table className="w-full">
+          <thead
+            className={clsx(
+              'h-[2.1875rem] bg-gray-50 rounded-t-lg border-b border-gray-200',
```

Comment:
You shouldn't set a fixed height here. The content of the cells should determine it. You can add padding inside each cell to make more space inside.

Reply:
this library not support horizontally mode so is hard to prepare it as on design